core/src/main/java/io/questdb/griffin/engine/table/LatestByValueListRecordCursor.java

```diff
@@ -135,15 +135,19 @@ public void of(DataFrameCursor dataFrameCursor, SqlExecutionContext executionCon
                 distinctSymbols++;
             }
             distinctSymbols = Math.max(0, distinctSymbols - excludedSymbolKeys.size());
-            rowIds.setCapacity(distinctSymbols);
+            if (distinctSymbols > 0) {
```

Comment:
I assume this is unrelated?

Reply:
I've added a couple of tests to cover this change: bc8c1d39abfab5840e2b893f3a7459ad3311cb37